lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,290 @@
+                                                           data-refund-info="<%- gettext('You will be refunded the amount you paid.') %>">
+                                                            <%- gettext('Unenroll') %>
+                                                        </a>
+                                                    <% }
+                                                } else {
+                                                    if (!model.is_course_blocked) { %>
+                                                        <a href="#unenroll-modal" class="action action-unenroll"
+                                                           rel="leanModal" data-course-id="<%- model.course_key %>"
+                                                           data-course-number="<%- model.course_number %>"
+                                                           data-course-name="<%- model.course_name %>"
+                                                           data-dashboard-index="<%- dashboard_index %>"
+                                                           data-cert-name-long="${cert_name_long | h}"
+                                                           data-track-info="<%- gettext('Are you sure you want to unenroll from the verified %(cert_name_long)s track of %(course_name)s (%(course_number)s)?') %>"
+                                                           data-refund-info="<%- gettext('The refund deadline for this course has passed, so you will not receive a refund.') %>">
+                                                            <%- gettext('Unenroll') %>
+                                                        </a>
+                                                    <% } else { %>
+                                                        <a class="action action-unenroll is-disabled"
+                                                           data-course-id="<%- model.course_key %>"
+                                                           data-course-number="<%- model.course_number %>"
+                                                           data-course-name="<%- model.course_name %>"
+                                                           data-dashboard-index="<%- dashboard_index %>"
+                                                           data-cert-name-long="${cert_name_long | h}"
+                                                           data-track-info="<%- gettext('Are you sure you want to unenroll from the verified %(cert_name_long)s track of %(course_name)s (%(course_number)s)?') %>"
+                                                           data-refund-info="<%- gettext('The refund deadline for this course has passed, so you will not receive a refund.') %>">
+                                                            <%- gettext('Unenroll') %>
+                                                        </a>
```

Comment:
There is a lot of repetition here. It seems like the only differences are in the href, class and rel attributes. If that is the case then these are the only things that should change. It would make sense to move this logic into the view so you are just passing these through into the templates (and would mean you can remove all of the if..else logic from this file)